src/inventory/gilded_rose.js

```diff
@@ -22,49 +22,80 @@ const items = [
 updateQuality(items);
 */
 export function updateQuality(items) {
+
   for (var i = 0; i < items.length; i++) {
-    if (items[i].name != 'Aged Brie' && items[i].name != 'Backstage passes to a TAFKAL80ETC concert') {
-      if (items[i].quality > 0) {
-        if (items[i].name != 'Sulfuras, Hand of Ragnaros') {
-          items[i].quality = items[i].quality - 1
+
+   //------VARIABLES-----//
+   
+    //create item variables to simplify & improve readability, decrease repetition, 
+    const item = items[i]
+    const agedBrie = ['Aged Brie'];
```

Comment:
If these variables are immutable and static (meaning they don't change) I'd consider moving them outside of this function and at the top of this file for easy editing!

Reply:
Okay I will move them. I think they were breaking and it goes to what we were discussing yesterday so I still have a bit of work to do on it.

Reply:
Just to be clear, mutable variables (i.e., const item = items[i]) should stay where it is since its value requires being placed in the forloop. However, moving agedBrie, backstagePass, and constantQuality outside of this function would not break the application. If it does, let me know because that might be highlighting a separate issue here.

Reply:
Yes, I learned that because when I moved const item it did break. I can see why as that is tied to the functionality in the for loop!

Reply:
The named variables work great in global scope :)